docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -1319,3 +1321,24 @@ _**default:**_ ""
 
 _References:_
 [http://nginx.org/en/docs/ngx_core_module.html#debug_connection](http://nginx.org/en/docs/ngx_core_module.html#debug_connection)
+
+## disable-pathtype-validation
+Ingress Controller validates the pathType, and only allows special characters on "path" if pathType is
+ImplementationSpecific. 
+
+The only characters allowed on ingresses with pathType not ImplementationSpecific 
+will be 0-9, a-z, A-Z, "-", ".", "_", "~", "/".
+
+If the validation is disabled, the [#path-additional-allowed-chars](#path-additional-allowed-chars) will
+be allowed on any pathType.
+
+This behavior can be disabled, so special characters are accepted regardless of pathType
+_**default:**_ "false"
+
+## path-additional-allowed-chars
+When validating path on Ingress resources, defines the additional set of special characters that
+will be allowed.
+
+See also [#disable-pathtype-validation](#disable-pathtype-validation).
+
+_**default:**_ "^%$[](){}*+?|"
```

Comment:
Please add \\ to this list.

Reply:
Its useful in escaping characters in regex. If a path contains a dot that needs not to be interpreted as a part of the regex, then it has to be escaped with backslash. If you think that's not common enough, you can skip it and users can override PathAdditionalAllowedChars if required.

Reply:
yeah, I agree right now if we can leave it on pathadditionalallowedchars and add as default on a followup, would be better :)

My concern with "\" is people abusing it to escape the escape ;P so unless we can say that "\" is the last character on configmap, etc, I would rather prefer to keep it out.

Thanks for reviewing this!